src/components/CategoryNavContainer/CategoryNav/CategoryNav.scss

```diff
@@ -1,12 +1,89 @@
 @import '~style/variables';
 
+.categoryNav {
+  background-color: $color-white;
+  display: flex;
+  padding-left: 20px;
+  width: 100%;
+  white-space: nowrap;
+  overflow-x: auto;
+  -webkit-overflow-scrolling: touch;
+  -ms-overflow-style: -ms-autohiding-scrollbar;
+
+  @include for-desktop-up {
+    padding-left: 30px;
+  }
+}
+
 .link {
+  height: 100%;
   display: inline-block;
-  padding: 10px;
+  padding: 12px 5px;
   text-decoration: none;
+
+  @include for-desktop-up {
+    padding: 24px 5px;
+  }
+
+  &:hover div {
+    color: $color-white;
+  }
+
+  div {
+    @include type-nav-horizontal;
+
+    padding: 0 24px;
+    border-radius: 100px;
+    color: $color-grey--700;
+    box-shadow: 0 0 4px 2px rgba(0, 0, 0, 0.1);
+
+    @include for-desktop-up {
+      box-shadow: initial;
+
+      &:hover {
+        box-shadow: 0 0 4px 2px rgba(0, 0, 0, 0.1);
+      }
+    }
+  }
+
+  &:hover,
+  &.link__active {
```

Comment:
This doesn't have different colours for the hover and active states. We have two options:
* do this task so that it can show different colours on hover, and just do, say `--500` for hover and `--600` for active. The header is then functional and we just update the colour values when we have them.
* We extend the task that updates the colours to add in functionality here.

Doesn't really matter, but I think it's better to lock down functionality in the task that describes it.

Reply:
Option A seems good, then we are just adding colour values in later.